src/main/java/se/liu/ida/hefquin/cli/modules/ModFederation.java

```diff
@@ -21,13 +21,14 @@ public void registerWith( final CmdGeneral cmdLine ) {
 	}
 
 	@Override
-	public void processArgs( final CmdArgModule cmdLine ) {
+    public void processArgs(final CmdArgModule cmdLine) {
 		if ( cmdLine.contains(fedDescrDecl) ) {
 			final String fedDescrFilename = cmdLine.getValue(fedDescrDecl);
 			cat = FederationDescriptionReader.readFromFile(fedDescrFilename);
+		} else {
```

Comment:
I prefer to have `else` in a new line. I know that this makes the code a little less compact, but I find it easier to read.

```suggestion
		}
		else {
```